scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,157 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-----------------------------------
+--Pre-requisite Completed Secrets of Oven Lost Log ID: 4, Quest ID: 73
+-----------------------------------
+-- Despachiaire !gotoid 16883721
+-- Rouva !gotoid 17719492
+--- Para NM ??? !gotoid 16785771
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+quest.reward =
+{
+    item     = xi.item.ELEGANT_RIBBON,
```

Comment:
Generally, compare your whitespacing to all the other quest files

Reply:
The section was copy pasted from another quest. So spacing seems off due to no other awards being granted. Updated.